examples/fluent-bit/fluent-bit.yaml

```diff
@@ -0,0 +1,227 @@
+---
```

Comment:
should we use helm chart instead https://docs.fluentbit.io/manual/installation/kubernetes#installing-with-helm-chart ?
which is easier to maintain

Reply:
yes, actually these lines are generated by `helm template`.